packages/fast-components-react-msft/src/caption/caption.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    CaptionLevel,
+    CaptionProps,
+    CaptionTag,
+    ICaptionHandledProps,
+    ICaptionManagedClasses,
+    ICaptionUnHandledProps
+} from "./caption";
+
+/**
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("caption snapshots", (): void => {
+    generateSnapshots(examples);
+});
+
+describe("caption", (): void => {
+    const Component: React.ComponentClass<ICaptionHandledProps> = examples.component;
+
+    test("should render a default `tag` of `CaptionTag.p` if no `tag` prop is passed", () => {
+        const rendered: any = shallow(
+            <Component />
+        );
+        const caption: any = rendered.first().shallow();
+
+        expect(caption.instance().props.tag).toEqual(CaptionTag.p);
+    });
+
+    test("should render the correct `level` when `level` prop is passed", () => {
```

Comment:
Can you add tests to ensure that all `tags` have the correct output?

Reply:
added examples for span, caption and figcaption